doc/classes/RayCast2D.xml

```diff
@@ -56,6 +56,21 @@
 			<return type="int" />
 			<description>
 				Returns the shape ID of the first object that the ray intersects, or [code]0[/code] if no object is intersecting the ray (i.e. [method is_colliding] returns [code]false[/code]).
+				To get the intersected shape node, use:
+				[codeblocks]
+				[gdscript]
+				var target = get_collider() # A CollisionObject2D or TileMap.
+				var shape_id = get_collider_shape() # The shape index in the collider.
+				var owner_id = target.shape_find_owner(shape_id) # The owner ID in the collider.
+				var shape = target.shape_owner_get_owner(owner_id)
+				[/gdscript]
+				[csharp]
+				var target = GetCollider(); // A CollisionObject2D or TileMap.
+				var shapeID = GetColliderShape(); // The shape index in the collider.
+				var ownerID = target.ShapeFindOwner(shapeID); // The owner ID in the collider.
+				var shape = target.ShapeOwnerGetOwner(ownerID);
```

Comment:
Yep, it needs a cast. I'm assuming it's meant to be a `CollisionObject2D` because I couldn't find the `Shape***` methods in `TileMap`.

Also minor nit-pick, the naming convention for ID is Id.

```suggestion
				var target = (CollisionObject2D)GetCollider(); // A CollisionObject2D or TileMap.
				var shapeId = GetColliderShape(); // The shape index in the collider.
				var ownerId = target.ShapeFindOwner(shapeId); // The owner ID in the collider.
				var shape = target.ShapeOwnerGetOwner(ownerId);
```